### DISCUSSION TITLE
[Windows] working with mobaXterm ?!

### DISCUSSION THREAD

**Discussion by tomitomas** [quexten/goldwarden#267] [Q&A]
Hello, 

[working on windows]

Does Goldwarden work with any "advanced" terminal like mobaXterm ? 
by "advanced", I mean not the standard "cmd" prompt nor PowerShell, but any other tools with more (graphic) option available.

---

which terminal do you use on your side ?

**Comment by tomitomas** [comment]
I found the perfect tool.

Thank you for all answers / ideas !  :-/

**Reply by quexten** [reply]
Sorry, I never got around to getting back to this. I will say that windows is untested at the moment. I do not have a windows device, so I don't know windows terminals. But I assume that mobaXTerm would work, as long as it is not sandboxed somehow (windows store app). Goldwarden - on windows - just overwrites the global named pipe, as long as you disable the openssh service on windows.

**Reply by quexten** [reply]
Reading https://docs.gcc.rug.nl/hyperchicken/ssh-agent-forwarding-mobaxterm/ makes it very unclear whether mobaxterm supports the openssh ssh-agent on windows. If it does, it supports interacting with goldwarden too, if not then not.

**Reply by tomitomas** [reply]
> Sorry, I never got around to getting back to this

no worries, I was expected answers also from others users than just you. 
It seems that I'm the only one still on windows ...! :sweat_smile:

> mobaxterm supports the openssh ssh-agent on windows. if it does, it supports interacting with goldwarden too, if not then not.

the answer is "no". Only Pageant, or their own agent, but not openssh service from windows. 
I got in touch with someone from the dev team, and they add it to their backlog, but no idea of when it's gonna come out --> that's why I created this thread

---

> I will say that windows is untested at the moment. I do not have a windows device, so I don't know windows terminals.

so I'll be the first one really testing it :) and I'll be pleased to help, if you want to (!?)

as far as now, it seems working pretty well (again thank you for the great job!)
there's some "minor issue" #269 , but it does not prevent the main functionality


I'm above all waiting for #265 to be released to confirm that most of it is OK on windows ! :)

thanks again @quexten
